packages/api/tests/lib/test-granules.js

```diff
@@ -91,3 +93,32 @@ test('getGranuleProductVolume() returns correct product volume', (t) => {
     0
   );
 });
+
+test('moveGranuleFilesAndUpdateDatastore throws if granulePgModel.getRecordCumulusId throws unexpected error', async (t) => {
+  const updateStub = sinon.stub().returns(Promise.resolve());
+  const granulesModel = {
+    update: updateStub,
+  };
+
+  const granulePgModel = {
+    getRecordCumulusId: () => {
+      const thrownError = new Error('Test error');
+      thrownError.name = 'TestError';
+      throw thrownError;
+    },
+  };
+
+  const collectionPgModel = {
+    getRecordCumulusId: () => 1,
```

Comment:
```suggestion
    getRecordCumulusId: async () => 1,
```

Reply:
👍🏻      That said, I don't think functionally it shouldn't matter in this case, as awaiting a non-async function is legit.  I went ahead and updated both stubs in this test: 3657713b9